pipelines/managed/rh-advisories/rh-advisories.yaml

```diff
@@ -884,27 +1084,3 @@ spec:
           workspace: release-workspace
       runAfter:
         - create-advisory
-  finally:
-    - name: cleanup
-      timeout: "4h"  # temp workaround until github.com/konflux-ci/release-service/issues/603 is fixed
-      taskRef:
-        resolver: "git"
-        params:
-          - name: url
-            value: $(params.taskGitUrl)
-          - name: revision
-            value: $(params.taskGitRevision)
-          - name: pathInRepo
-            value: tasks/managed/cleanup-workspace/cleanup-workspace.yaml
```

Comment:
This task doesn't only cleanup the workspace. It also cleans up internalRequests created by this pipelinerun. I think we still need that

Reply:
added new cleanup-internal-requests task